dev/tidb-in-kubernetes/faq.md

```diff
@@ -20,3 +20,60 @@ The default time zone setting for each component container of a TiDB cluster in
 
     * In the `values.yaml` file of the TiDB cluster, modify `timezone` settings in the `values.yaml` file of the TiDB cluster. For example, you can set it to `timezone: Asia/Shanghai` and then upgrade the TiDB cluster.
     * Refer to [Time Zone Support](/how-to/configure/time-zone.md) to modify TiDB service time zone settings.
+
+## Can HPA or VPA be configured on TiDB components?
+
+Currently, TiDB cluster does not support HPA (Horizontal Pod Autoscaling) or VPA (Vertical Pod Autoscaling), because it is difficult to achieve autoscaling on applications with state such as database. Autoscaling can not be achieved merely by the monitoring data of CPU and memory.
```

Comment:
```suggestion
Currently, the TiDB cluster does not support HPA (Horizontal Pod Autoscaling) or VPA (Vertical Pod Autoscaling), because it is difficult to achieve autoscaling on stateful applications such as a database. Autoscaling can not be achieved merely through the monitoring data of CPU and memory.
```